app/main.ipynb

```diff
@@ -496,21 +636,34 @@
     "lst = [1, 2, 3, 4, 5]"
    ],
    "metadata": {
-    "collapsed": false
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2023-11-14T13:20:26.392769600Z",
+     "start_time": "2023-11-14T13:20:26.199213700Z"
+    }
    }
   },
   {
    "cell_type": "code",
-   "execution_count": 28,
-   "outputs": [],
+   "execution_count": 21,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "3.0"
+     },
+     "execution_count": 21,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "# write your numpy code here and estimate the time of execution"
+    "np.array(lst).mean()"
```

Comment:
i think, it will be better to use np.average(lst) for this task, but your solution works too